QuickDeployment/QuickDeploymentDesktopReadme.md

```diff
@@ -0,0 +1,155 @@
+# Thanks for trying Chocolatey For Business!
+
+This system has been pre-configured as a fully functioning C4B environment.
+
+## Create a license package
+
+___
+
+To leverage all of the features of C4B, copy the license file you received via email to C:\ProgramData\chocolatey\license. Make sure the name of the file is exactly "chocolatey.license.xml"
+
+In Powershell execute:
+
+```powershell
+Set-ExecutionPolicy Bypass -Scope Process -Force ; . 'C:\choco-setup\files\CreateLicensePackage.ps1'
+```
+
+This will create the licensed package at C:\choco-setup\packages and push it up to your Nexus repository for use.
+
+## Enable Central Management
+
+___
+
+**NOTE:** This step should _only_ be completed once the license package has been created in the step above. All licensed features are already installed, just unactivated without a valid license file.
+
+Run the following to turn on the Central Management services.
+
+```powershell
+Set-ExecutionPolicy Bypass -Scope Process -Force ; . 'C:\choco-setup\files\EnableCCM.ps1'
+```
+
+## Server Information
+
+___
+
+### Nexus Repository
+
+- Url: [https://chocoserver:8443](https://chocoserver:8443)
+- Username: admin
+- Password: [REDACTED]
+- API Key: [REDACTED]
+
+When you first log in to Nexus, you will immediately be asked you change your password.
+You will then be asked if you'd like to enable Anonymous Access to the repositories. We typically recommend doing this, unless security requirements in your organization stipulate that RBAC controls be in place.
+
+Sources configured in chocolatey can only read data from their remote endpoints, and cannot delete items. If you need to limit the packages people have access to, control this with separate Hosted and Group repositories. Reach out to Chocolatey Support or consult the Nexus documentation for more information.
+
+#### **Changing the API Key**
+
+You may wish to change the API key before you start using things. To do so, log in to Nexus using the information above, or you rnew credentials if you have already gone through the first run wizard. Once logged in perform the following steps:
+
+1. Click on your username in the upper right-hand side of the homepage.
+2. Select "NuGet API Key" from the left-hand navigation window.
+3. Select "Reset API key"
+4. Enter your password
+5. Take note of the new API key
+
+If you change your API key, you will need to change the key in the Jenkins jobs that are pre-configured for you. See the next section for information on how to connect to Jenkins.
+
+#### Choco apikey Command
+
+To help make pushing packages easier, the `choco apikey` command is available. This will store your API key for a specific source as part of chocolatey's configuration. This will be encrypted. To setup, do the following:
```

Comment:
```suggestion
To help make pushing packages easier, the `choco apikey` command is available. This will store your API key for a specific source as part of Chocolatey's configuration. This will be encrypted. To setup, do the following:
```